hikari/impl/rest.py

```diff
@@ -357,21 +310,6 @@ def __init__(
     ) -> None:
         self._http_settings = config.HTTPSettings() if http_settings is None else http_settings
         self._proxy_settings = config.ProxySettings() if proxy_settings is None else proxy_settings
-
-        # Lazy initialized later, since we must initialize this in the event
-        # loop we run the application from, otherwise aiohttp throws complaints
-        # at us. Quart, amongst other libraries, causes issues with this by
-        # making a new event loop on startup, which means if we initialised
-        # the connector here and initialised this class in global scope, it
-        # would potentially end up using the wrong event loop and aiohttp
-        # would then fail when creating an HTTP request.
-        if connector_factory is None:
-            connector_factory = BasicLazyCachedTCPConnectorFactory(self._http_settings)
-            connector_owner = True
-
-        self._connector_factory = connector_factory
-        self._connector_owner = connector_owner
-        self._event_loop: typing.Optional[asyncio.AbstractEventLoop] = None
         self._executor = executor
```

Comment:
do executors guarantee thread safety?

Reply:
Generally, yes:

https://stackoverflow.com/questions/45286344/is-python-threadpoolexecutor-thread-safe